react/OrderItems.tsx

```diff
@@ -65,11 +65,12 @@ const updateTotalizersAndValue = ({
   const oldItemSellingPrice = oldItem.sellingPrice ?? 0
 
   const oldPrice = oldItemPrice * oldItemQuantity
-  const newPrice = newItem.price * newItem.quantity
+  const newPrice = newItem.price! * newItem.quantity
```

Comment:
Why is this non-null assertion safe?

Reply:
that's because this item has an availability of `available`. we changed the types on the GraphQL schema because the price could be `null` if the store owner didn't register the price yet for that particular item (has an availability of `withoutPriceFullfilment`), so this is safe